questions.md

```diff
@@ -0,0 +1,5 @@
+# Some questions about pull requests:
+
+*  what happens with multiple branches?
+*  how do you resolve merge conflicts?
+*  what if code reviewers don't approve code?
```

Comment:
You can set your preferences on how you want GitHub to act when it comes to code reviews. As an administrator you can protect your master branch and require someone on the team to approve before merging or you can just allow anyone to merge even if someone does not approve if you do not set restrictions. More on that subject [here](https://help.github.com/en/articles/enabling-required-reviews-for-pull-requests).